src/plugins/intel_cpu/tests/functional/single_layer_tests/activation.cpp

```diff
@@ -178,7 +177,28 @@ const auto basicCases3D = ::testing::Combine(
 
 INSTANTIATE_TEST_SUITE_P(smoke_Activation3D_Eltwise_CPU_BF16, ActivationLayerCPUTest, basicCases3D, ActivationLayerCPUTest::getTestCaseName);
 
-/* ============= Activation (2D) ============= */
+const std::map<ActivationTypes, std::vector<std::vector<float>>> activationTypes_blocked = {
+        {Mish,        {{}}},
+        {SoftSign,    {{}}}
```

Comment:
I think we should create ticket to add these ops to snippets. It should be in another PR, sure. Could you please create the ticket? Thanks!

Reply:
Good point, thanks. 99811 and 99812 are created